packages/nimble-components/docs/component-usage-patterns/combobox-select-menu.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as selectStories from '../../src/select/tests/select.stories';
+import * as comboboxStories from '../../src/combobox/tests/combobox.stories';
+import * as menuButtonStories from '../../src/menu-button/tests/menu-button.stories';
+
+<Meta title="Usage/Combobox, Select, & Menu Button"/>
+
+# Combobox, Select, & Menu Button Usage
+
+### What's the same?
+The `nimble-select`, `nimble-combobox`, and `nimble-menu-button` are all Nimble components used for presenting a list 
+of options to the user. 
+
+### What's the difference?
+`nimble-select` is used for simple selection tasks with predefined options, 
+`nimble-combobox` is used when there are many options or users need to search for a specific value, and 
+`nimble-menu-button` is used for presenting a list of options, commands, or links to the user.
```

Comment:
The word "options" is too general to provide meaningful contrast about the menu-button versus select and combobox. Actions, commands, or links--maybe?